frontend/app/routes/protected/index.tsx

```diff
@@ -28,12 +30,24 @@ export default function Index() {
 
   return (
     <div className="mb-8">
-      <Menu>
-        <MenuItem to="/">{t('protected:index.home')}</MenuItem>
-        <MenuItem file="routes/protected/admin.tsx">{t('protected:index.admin-dashboard')}</MenuItem>
-        <MenuItem file="routes/public/index.tsx">{t('protected:index.public')}</MenuItem>
-        <MenuItem file="routes/protected/person-case/first-name.tsx">{t('protected:index.in-person')}</MenuItem>
-      </Menu>
+      <h1 className="mt-8 text-2xl font-bold text-slate-700">
+        <span className="underline decoration-red-800 underline-offset-8">{t('protected:dashboard.sin')}</span>{' '}
+        {t('protected:dashboard.system')} {t('protected:index.dashboard')}
```

Comment:
Is there any reason this string (SIN System Dashboard) is broken into three separate i18n resources? Couldn't you make it a single string `<span>SIN</span> System Dashboard` and then use `<Trans>` to replace the span with the styled one?